pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.poetry]
+name = "kiwi"
+version = "10.0.0"
+description = "KIWI - Appliance Builder"
+license = "GPL-3.0-or-later"
+readme = "README.rst"
+homepage = "https://osinside.github.io/kiwi/"
+documentation = "https://osinside.github.io/kiwi/"
+repository = "https://github.com/OSInside/kiwi"
+
+authors = [
+   "Marcus Schäfer <ms@suse.com>",
```

Comment:
This needs to change to ```marcus.schaefer@suse.com``` my former address I didn't get back

Reply:
That sucks. 😦